addons/binding/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/handler/BigAssFanHandler.java

```diff
@@ -565,6 +554,8 @@ private ThingStatusDetail getDetail() {
         private final long FAN_LISTENER_DELAY = 2L;
```

Comment:
Can the FanListener class get its own file?

Reply:
I was going to do that in a separate PR. Ok?